modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/KotlinVertxServerCodegen.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.openapitools.codegen.languages;
+
+import org.openapitools.codegen.CodegenConstants;
+import org.openapitools.codegen.CodegenType;
+import org.openapitools.codegen.SupportingFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.Locale;
+
+public class KotlinVertxServerCodegen extends AbstractKotlinCodegen {
+
+    protected String rootPackage = "org.openapitools.server.api";
+    protected String apiVersion = "1.0.0-SNAPSHOT";
+
+    public static final String ROOT_PACKAGE = "rootPackage";
+
+    public static final String PROJECT_NAME = "projectName";
+
+    static Logger LOGGER = LoggerFactory.getLogger(KotlinVertxServerCodegen.class);
+
+    public CodegenType getTag() {
+        return CodegenType.SERVER;
+    }
+
+    public String getName() {
+        return "kotlin-vertx";
```

Comment:
Heads up: I'll rename it as "kotlin-vertx-server" to follow the naming convention.

Reply:
Second though: kotlin-vertx is fine given that we've kotlin-spring